server/room-battle.ts

```diff
@@ -726,6 +726,10 @@ export class RoomBattle extends RoomGames.RoomGame {
 		for (const player of this.players) player.wantsTie = false;
 
 		switch (lines[0]) {
+		case 'log':
+			if (!this.logData) this.logData = {};
+			this.logData.inputLog = lines.slice(1);
```

Comment:
So we have a `battle.inputLog` and a `battle.logData.inputLog` which are usually out of sync? That seems pretty ugly tbh...